js-exercises/time-was/timeWas.js

```diff
@@ -0,0 +1,46 @@
+
+const isValidDate = (dateInMillieconds) => {
+  return typeof dateInMillieconds === 'number' && !Number.isNaN(dateInMillieconds);
+};
+
+function timeWas(timeInMilliseconds, currentDateInMilliseconds) {
+  if (!isValidDate(timeInMilliseconds)) {
+    throw new Error('please enter time in milliseconds');
+  }
+  // if current date is given , override the default Date.now()
+  if (currentDateInMilliseconds) {
+    if (!isValidDate(currentDateInMilliseconds)) {
+      throw new Error('please enter current date  in milliseconds');
+    }
+    Date.now = () => currentDateInMilliseconds;
+  }
+
+  const timeInSeconds = Math.floor((Date.now() - timeInMilliseconds) / 1000);
+
+  if (timeInSeconds <= 1) {
+    return 'just now';
+  }
+
+  // values in seconds
+  const intervals = {
```

Comment:
How about your remove the comment and name the variable like this? 
```suggestion
  const valuesInSeconds = {
```

Reply:
changed intervals to valuesInSeconds.